src/components/registerV2/confirmPassphrase.css

```diff
@@ -32,10 +34,9 @@
   width: 100%;
 
   & .word {
-    color: var(--color-content);
-    font-family: var(--heading-font);
-    font-size: inherit;
-    font-weight: var(--font-weight-bold);
+    @mixin headingSmall;
+
+    color: var(--color-maastricht-blue);
     letter-spacing: 0.1px;
     line-height: var(--word-choice-height);
```

Comment:
you can remove the letter-spacing and line-height, since they are already set on the mixin